.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  - push
+  - pull_request
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install --global awesome-lint
+      - run: awesome-lint
```

Comment:
Could just use `npx awesome-lint` per the docs, seem redundant having multiple steps.